src/services/textChanges.ts

```diff
@@ -339,6 +339,7 @@ namespace ts.textChanges {
             this.deletedNodes.push({ sourceFile, node });
         }
 
+        /** Stop! Consider using `delete` instead, which has logic for deleting nodes from delimited lists. */

```

Comment:
It’s the method declaration right before this one

Reply:
arghhh. I'm pretty sure I was searching for `public delete` in my editor instead of just `delete`. I saw all the `public` keywords and assumed that it was OK to swap in `public xxx` for my usual `function xxx` search habits.